docs/getting-started/requirements.md

```diff
@@ -71,14 +71,23 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
 
 #### Azure Active Directory scope
 
-Azure Active Directory (AAD) permissions required by the tool are the following:
+Microsoft Entra ID (AAD earlier) permissions required by the tool are the following:
 
 - `Directory.Read.All`
 - `Policy.Read.All`
 
 The best way to assign it is through the azure web console:
 
-![AAD Permissions](../img/AAD-permissions.png)
+1. Access to Microsoft Entra ID
+2. Go to "App registrations" and register a new application
+  ![Register an Application page](../img/register-application.png)
+3. Once your application is registered, select "API permissions" blade
+4. Add the needed permissions:
+    - `Directory.Read.All`
+    - `Policy.Read.All`
+
+    ![EntraID Permissions](../img/AAD-permissions.png)
+
 
 #### Subscriptions scope
 
```

Comment:
Review the double space after `subscription`

Reply:
Done